scripts/01_deploy_pillar_staking.js

```diff
@@ -0,0 +1,39 @@
+const { ethers, run } = require("hardhat");
+
+const main = async () => {
+  await run("compile");
+
+  const stakingToken = "0xe3818504c1B32bF1557b16C238B2E01Fd3149C17"; // add staking token address here (PLR)
```

Comment:
These contract addresses are valid only on Ethereum Mainnet. So please add a necessary check to validate the chainId and then use these addresses.

Reply:
Done